client/src/components/Upload.vue

```diff
@@ -229,10 +289,24 @@ function prepareFiles(files) {
                 />
               </v-col>
             </v-row>
-            <v-list-item-subtitle
-              v-if="pendingUpload.type === 'image-sequence'"
-            >
-              {{ pendingUpload.files.length }} images
+            <v-list-item-subtitle>
+              {{ computeUploadProgress(pendingUpload) }}
+              <!-- errorMessage is provided by the fileUploader mixin -->
```

Comment:
While I appreciate this, I think it's a lots cause.   I've never seen anyone comment every time they use a mixin feature.  Is this considered a best practice, or just something you like to do?

Reply:
It's something I do just for ease of use and readability if I ever have to look at it again.  Is there an extension for VS code or some setup that will goTo a function that is imported as a mixin from a node_module.  If I came at this with standard VS code setup and ctrl+shft+F for the function name I wouldn't be able to find it without turning on searching inside of node_modules.  Then if I do something like 'errorMessage' is found 400+ times.  This is the part where I really wish there was some namespacing for mixins, or even a naming standard mixinErrorMessage.  I can obviously take some of them out.